osf/models/spam.py

```diff
@@ -221,6 +224,13 @@ def do_check_spam(self, author, author_email, content, request_headers, update=T
         if self.is_spammy:
             return True
 
+        enqueue_task(
```

Comment:
Wondering if this should be configurable via settings since so many of our SPAM things are. Thoughts @mfraezz?